envoy/config/subscription_factory.h

```diff
@@ -13,6 +13,10 @@ class SubscriptionFactory {
 public:
   virtual ~SubscriptionFactory() = default;
 
+  // Default Parameters for the backoff strategy.
+  static constexpr uint32_t RetryInitialDelayMs = 500;
+  static constexpr uint32_t RetryMaxDelayMs = 30000; // Do not cross more than 30s
```

Comment:
What is this comment intending to convey? If it's just "this is the max", then not needed (it's in the name). If there's some reason the max can't be more than 30s, it would be good to explain why.

Reply:
The intent of the PR was to not change the legacy values and behavior from https://github.com/envoyproxy/envoy/blob/main/source/common/config/grpc_stream.h#L22

Unfortunately, I am not sure why 30s was chosen to be the maximum. I can remove the comment here if that is okay.

Reply:
I think just delete the comment, yes, seems a bit confusing.